packages/components/src/components/event/eventContent/EventContent.tsx

```diff
@@ -26,12 +34,28 @@ interface Props {
   event: EventFields;
   superEvent?: SuperEventResponse;
   hasSimilarEvents?: boolean;
+  appConfig: AppConfig;
+  getPlainEventUrl: (event: EventFields, locale: AppLanguage) => string;
+  iconDirections: ElementType;
+  getOrganizationSearchUrl: GetOrganizationSearchUrl;
+  useSubEvents: UseSubEvents;
+  useSubEventsQueryVariables: UseSubEventsQueryVariables;
+  useOtherEventTimes: UseOtherEventTimes;
```

Comment:
It's not a good habit to add hooks as props. The getters and setters that it offers should be used instead.